### PR TITLE
Update locked dependencies to their latest

### PR DESCRIPTION
**Pull request by warmwaffles**
* Moves credo config to where it usually is (top level of project).
* Fixes one credo issue.
* Remove unnecessary commented code.

### CODE REVIEW

File: test/ex_image_info_test/images/bmp_test.exs [hunk 1]
```diff
@@ -14,7 +14,7 @@ defmodule ExImageInfoTest.Images.BMPTest do
   test "force - bmp disk image - #seems? #type #info", images do
     assert seems?(images["bmp"], :bmp) == true
     assert type(images["bmp"], :bmp) == {"image/bmp", "BMP"}
-    assert info(images["bmp"], :bmp) == {"image/bmp",130, 42, "BMP"}
+    assert info(images["bmp"], :bmp) == {"image/bmp", 130, 42, "BMP"}
   end
 
   # @tag :wip
```
Comment (rNoz) [inline test/ex_image_info_test/images/bmp_test.exs]: Maybe include the removal of these unneeded comments?
```suggestion

```

**Review by rNoz — APPROVED** [review]
Thanks  for this!

File: test/ex_image_info_test/images/bmp_test.exs (hunk 1 above)
Comment (warmwaffles) [reply test/ex_image_info_test/images/bmp_test.exs]: Yea we can knock those out too

Comment (warmwaffles) [reply test/ex_image_info_test/images/bmp_test.exs]: Done.

**Pull request merged**